packages/components/card/src/card.story.js

```diff
@@ -1,41 +1,61 @@
 /* eslint-disable react/prop-types */
 
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text } from '@storybook/addon-knobs/react';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs/react';
 import Spacings from '@commercetools-uikit/spacings';
 import Text from '@commercetools-uikit/text';
 import Section from '../../../../docs/.storybook/decorators/section';
 import Readme from '../README.md';
 import Card from './card';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 const themes = ['light', 'dark'];
 const insetScaleValues = ['none', 's', 'm', 'l', 'xl'];
 const content =
   'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Arcu dictum varius duis at consectetur lorem donec.';
 
-const CardStoryTemplate = (props) => (
-  <Section>
-    {themes.map((theme) => (
-      <Spacings.Stack key={theme}>
-        <Text.Headline as="h2">
-          <code>theme: {theme}</code>
-        </Text.Headline>
-        <Spacings.Inline>
-          {insetScaleValues.map((scale) => (
-            <Spacings.Stack key={scale} scale="s">
-              <Text.Body fontWeight="bold">
-                <code>insetScale: {scale}</code>
-              </Text.Body>
-              <Card insetScale={scale} type={props.type} theme={theme}>
-                {text('content', content)}
-              </Card>
-            </Spacings.Stack>
-          ))}
-        </Spacings.Inline>
-      </Spacings.Stack>
-    ))}
-  </Section>
-);
+const CardStoryTemplate = (props) => {
+  const onClickEnabled = boolean('onClick enabled', false);
+  const isDisabled = boolean('isDisabled', false);
+  const to = text('to', '');
+  const isExternalLink = boolean('isExternalLink', false);
```

Comment:
This property does not change anything in the components UI nor in its behaviour inside the story so I think it's not needed and we can use just a fixed value.

Reply:
toggling this as `true` opens a new tab in the story if `to` is set - still ok to remove?
![2024-01-30 11 13 50](https://github.com/commercetools/ui-kit/assets/15024562/b5e44535-d7a8-43d5-8db8-a74fb518b899)

Reply:
The way I see it, it doesn't add any value and it can even lead to thinking the story is broken.

Let's see what @chloe0592 and @kark think of it.

Reply:
Sounds good - for what it's worth, this behavior exists for the `Link` component as well
![2024-01-30 11 18 21](https://github.com/commercetools/ui-kit/assets/15024562/786338d1-5703-455d-889b-ef40a5a21a87)

Reply:
@CarlosCortizasCT @chloe0592 Glad we reached consensus, addressed in 2b97618de31a7ab2ea37d3ee40dba8a5bd2848f5